### DISCUSSION TITLE
How not to skip "Repository is a mirror - skipping"

### DISCUSSION THREAD

**Discussion by ankitdadhich** [renovatebot/renovate#16327] [Request Help]
How can we not skip `Repository is a mirror - skipping`

Regardless of repository is mirror or not, can it create onboarding merge request and later update dependencies?

`11 0.429 {"name":"renovate","hostname":"buildkitsandbox","pid":1,"level":20,"logContext":"kM4xtiP9c25LZebpcWZ84","repository":"loca-path-to-repo","msg":"Repository is a mirror - throwing error to abort renovation","time":"2022-06-29T09:06:05.789Z","v":0}
#11 0.429 {"name":"renovate","hostname":"buildkitsandbox","pid":1,"level":20,"logContext":"kM4xtiP9c25LZebpcWZ84","repository":"loca-path-to-repo","err":{"message":"mirror","stack":"Error: mirror\n    at Proxy.initRepo (/app/node_modules/renovate/dist/modules/platform/gitlab/index.js:169:19)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async getPlatformConfig (/app/node_modules/renovate/dist/workers/repository/init/apis.js:36:28)\n    at async initApis (/app/node_modules/renovate/dist/workers/repository/init/apis.js:45:14)\n    at async initRepo (/app/node_modules/renovate/dist/workers/repository/init/index.js:29:14)\n    at async Object.renovateRepository (/app/node_modules/renovate/dist/workers/repository/index.js:37:18)\n    at async Object.start (/app/node_modules/renovate/dist/workers/global/index.js:120:13)\n    at async /app/node_modules/renovate/dist/renovate.js:15:24"},"msg":"Caught initRepo error","time":"2022-06-29T09:06:05.790Z","v":0}`

**Comment by PhilipAbed** [comment]
in a case where a Mirrored repository is also considered a fork, 
i would try this [includeforks](https://docs.renovatebot.com/configuration-options/#includeforks)

if not then i dont think we support any flag to include mirrored repos.

@rarkins @viceice please confirm

**Reply by rarkins** [reply]
I think we have a hardcoded rule to skip mirrored repositories. @ankitdadhich can you explain your use case and why you'd run Renovate on the mirror and not the source repo?

**Reply by ankitdadhich** [reply]
My bad for late reply. My org have github and gitlab both as of now. We are moving towards gitlab fully. But it may take a while.
So there are some repos which are on both with same name, but renovate treats them as mirror. 
Actually the particular repos are not duplicate on gitlab itself.

**Reply by rarkins** [reply]
I guess we'd need a new option "includeMirrors"

**Comment by PhilipAbed** [comment]
@ankitdadhich you can open a new issue ( feature request) to support `includeMirrors`, we currently don't support it.

**Reply by ankitdadhich** [reply]
https://github.com/renovatebot/renovate/issues/16436
